app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -58,116 +80,99 @@ class ReceiveModal extends React.Component {
       return null
     }
 
-    let qrCode
+    let qrCode, message
     switch (qrCodeType) {
       case QRCODE_TYPE_ADDRESS:
         qrCode = address
+        message = intl.formatMessage({ ...messages.copy_address })
         break
       case QRCODE_TYPE_PUBKEY:
         qrCode = pubkey
+        message = intl.formatMessage({ ...messages.copy_pubkey })
+        break
+      case QRCODE_TYPE_LNDCONNECT:
+        qrCode = lndConnect
+        message = intl.formatMessage({ ...messages.copy_uri })
         break
     }
 
     return (
       <Modal onClose={closeReceiveModal}>
-        <Flex justifyContent="center" alignItems="center" css={{ height: '100%' }}>
-          <Flex
-            justifyContent="space-between"
-            alignItems="center"
-            width={1}
-            bg="secondaryColor"
-            p={4}
-          >
-            <Flex as="section" flexDirection="column" alignItems="center" width={1 / 3}>
-              <Heading.h3 textAlign="center">
-                {alias && alias.length ? alias : pubkey.substring(0, 10)}
-              </Heading.h3>
-              <Flex justifyContent="center" mb={3}>
+        <Flex
+          justifyContent="space-between"
+          alignItems="center"
+          width={1}
+          bg="secondaryColor"
+          p={4}
+        >
+          <Flex as="section" flexDirection="column" alignItems="center" width={1}>
+            <Heading.h1 textAlign="center">
+              {alias && alias.length ? alias : pubkey.substring(0, 10)}
+              {network && network.name.toLowerCase() === 'testnet' && ` (${network.name})`}
```

Comment:
Looks like it can be simplified to this
` {network && network.name.toLowerCase() === 'testnet' && network.name}`

Reply:
Network name is supposed to be displayed in brackets after the node alias.

Reply:
I've updated this to use the `WalletName` component, so that the name displayed here matches elsewhere in the wallet (eg in the settings menu).